Application/EdFi.Ods.Standard/Artifacts/MsSql/Structure/Ods/1301-CreateEdOrgToEdOrgTriggers.sql

```diff
@@ -0,0 +1,24 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TRIGGER edfi.edfi_EducationOrganization_TR_Insert ON edfi.EducationOrganization AFTER INSERT AS
+BEGIN
+    SET NOCOUNT ON
+    INSERT INTO auth.EducationOrganizationIdToEducationOrganizationId
+    SELECT i.EducationOrganizationId, i.EducationOrganizationId
+    FROM inserted as i
+END
+GO
+
+CREATE TRIGGER edfi.edfi_EducationOrganization_TR_Delete ON edfi.EducationOrganization AFTER DELETE AS
+BEGIN
+    SET NOCOUNT ON
+    DELETE auth.EducationOrganizationIdToEducationOrganizationId
+    FROM auth.EducationOrganizationIdToEducationOrganizationId
+    INNER JOIN deleted d
+        ON SourceEducationOrganizationId = d.EducationOrganizationId
+        AND TargetEducationOrganizationId = d.EducationOrganizationId
```

Comment:
This will only catch the entry created by the insert trigger.  This delete trigger should clean up any entries that reference the Education Organization Id, since this is the consistent entry point for a delete for any given sub class.  For example I manually created a LEA -> School entry here like we would in a future ticket, and this delete trigger didn't clean it up.  Probably just doing an OR instead of an AND should be enough.